shelter/shelter-shelter-interview.vbs

```diff
@@ -101,23 +161,29 @@ EMWriteScreen MAXIS_case_number, 18, 43
 EMWriteScreen CM_mo, 20, 43	'entering current footer month/year
 EMWriteScreen CM_yr, 20, 46
 
+CALL determine_program_and_case_status_from_CASE_CURR(case_active, case_pending, case_rein, family_cash_case, mfip_case, dwp_case, adult_cash_case, ga_case, msa_case, grh_case, snap_case, ma_case, msp_case, emer_case, unknown_cash_pending, unknown_hc_pending, ga_status, msa_status, mfip_status, dwp_status, grh_status, snap_status, ma_status, msp_status, msp_type, emer_status, emer_type, case_status, list_active_programs, list_pending_programs)
+
+when_contact_was_made = date & ", " & time
+IF name_verf <> "" THEN homeless_verified = "YES"
+
 'The case note'
 start_a_blank_CASE_NOTE
-Call write_variable_in_CASE_NOTE(">>>DO NOT release " & cash_type & " funds, family in shelter<<<")
-Call write_variable_in_CASE_NOTE("* 100% of cash benefit to be issued to HCEA shelter account #52871.")
-If cash_type = "DWP" then
-    Call write_variable_in_CASE_NOTE("* DWP families are not eligible for the one-time only personal needs and medical co-pays")
-ELSE
-    Call write_variable_in_CASE_NOTE(" Except $" & one_time_issuance & " to EBT for one-time only (10%) for personal needs. $20 for medical co-pays.")
-END IF
-Call write_variable_in_CASE_NOTE("---")
-Call write_bullet_and_variable_in_CASE_NOTE("Other income", other_income)
-Call write_bullet_and_variable_in_CASE_NOTE("Money mismanagement", money_mismanagement)
-Call write_bullet_and_variable_in_CASE_NOTE("Reason for homelessness", reason_homeless)
+CALL write_variable_in_CASE_NOTE("### Family in Shelter Interview with M" &  MEMB_number & " ####")
```

Comment:
Is this only for family cases? Was this script for anything other than self-pay issues?

Reply:
Interviews are done on all family cases looking to shelter